File: bulochka/Entities/Bulochka.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
namespace bulochka.Entities

{
    public class Bulochka
    {
        public long Id { get; set; }
        public BulochkaTypes Type { get; set; }
        public DateTime CreationTimestamp { get; set; }
        public DateTime LastupdateTimestamp {get; set;}
        public DateTime DropTimestamp { get; set; }
        public DateTime ControlSellTime { get; set; }
        [Range(0,100)]
        public float StartPrice { get; set; }
        public float CurrentPrice { get; set; }
    }
}

File: bulochka/ClientApp/src/app/shared/models/bulochka.ts
import { BulochkaTypes } from "./bulochkaTypes";

export interface IBulochka {
    id: number;
    creationTimestamp: string;
    lastupdateTimestamp: string;
    dropTimestamp: string;
    startPrice: number;
    currentPrice: number;
    controlSellTime: string;
    type: BulochkaTypes;
} 